source/standards/dns-hosting.html.md.erb

```diff
@@ -1,33 +1,58 @@
 
-For example, `publishing.service.gov.uk` is published to both Amazon Route 53 and Google Cloud DNS to remove vendor as a single point of failure.
-This means that even if AWS has a total outage, users can access parts of GOV.UK by using Google Cloud DNS to resolve addresses, and viewing content that is available in the CDN cache.
+[Service Manual]: https://www.gov.uk/service-manual/technology/get-a-domain-name#choose-where-youll-host-your-dns
+[Amazon Route 53]: https://aws.amazon.com/route53/
+[Google Cloud DNS]: https://cloud.google.com/dns/
+[service domains and DNS]: https://www.gov.uk/service-manual/technology/get-a-domain-name
 
-Note that if your service availability relies on a single vendor, there is less benefit to deploying DNS to multiple vendors.
+## Production and non-production domain names
 
-When you implement your DNS strategy you should consider using:
+Your non-production domains should be subdomains of your production domain.
 
-* [Amazon Route 53][] as a cloud DNS web service
-* [Google Cloud DNS][] for high nameserver availability
+For example, if you service can be found at `myservice.service.gov.uk`, then you might also have Route 53 records for your other environments, such as:
 
-You can read more about [service domains and DNS][] in the Service Manual.
+* `staging.myservice.service.gov.uk`
+* `dev.myservice.service.gov.uk`
 
-[Service Manual]: https://www.gov.uk/service-manual/technology/get-a-domain-name#choose-where-youll-host-your-dns
-[Amazon Route 53]: https://aws.amazon.com/route53/
-[Google Cloud DNS]: https://cloud.google.com/dns/
-[service domains and DNS]: https://www.gov.uk/service-manual/technology/get-a-domain-name
+You can manage all of these records from your production Route 53 configuration pipeline, even though they relate to other environments.
+
+If your service has more complex needs you may alternatively
+[delegate](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/dns-routing-traffic-for-subdomains.html)
+the zone for a subdomain to an AWS account that relates specifically to that environment.
+
+## Internal GDS services
+
+Internal GDS services must be hosted on subdomains of the `digital.cabinet-office.gov.uk` domain.
+
+> [!TIP]
+> This domain is likely to change as part of GDS moving to DSIT.
+
+Once you have
+[configured your Route 53 hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html),
+you'll need to use the
+[GDS IT Help Desk](https://gdshelpdesk.digital.cabinet-office.gov.uk/helpdesk)
+to request delegation of your subdomain - choose:
+
+* IT Requests
+* Changes - Request & Support
+* DNS
+
+You must follow the [GDS brand guidance][] for services that are accessible by the public but which don't use the GOV.UK brand.
+
+[GDS brand guidance]: https://docs.google.com/document/d/1MXFXP7oMxALiA4t6Gw8XFPksMI-aBjKqO5wCHvnccyo
```

Comment:
Turns out this is a link to _my own Google drive_ because we received this guidance by _email_. Gaaaaah.

Reply:
No it wasn't, George confused is all